### DISCUSSION TITLE
Question about the student package

### DISCUSSION THREAD

**Discussion by goldn-apk** [community/community#123492] [GitHub Education] [answered]
### Select Topic Area

Question

### Body

Dear GitHub Support Team,

I recently applied for the GitHub Student Developer Pack, which was approved, but my student account hasn't been activated after four days. Despite receiving a message to wait for activation, I'm still unable to access the benefits. Could you please investigate and assist with activating my account promptly? .Thank you for your prompt attention to this matter.

**Comment by davevad93** [comment]
Hi @isbani1 ,

If your application request has a green badge and a message saying "`Congrats! Your request for @isbani1 was approved and your benefits will be available soon.`" it means that you have to wait, that's why your benefits aren't yet available.

![3](https://github.com/community/community/assets/94308234/49d7f2a3-c3cb-4632-9e6e-a1bed3f6576e)

If your application request has a purple badge and a message saying "`Congrats! Your request for @isbani1 was approved and your benefits are now available.`" it means that your benefits should be available, so if you can't access them you should contact [GitHub Support](https://support.github.com/request/education)

![2](https://github.com/community/community/assets/94308234/ae64cf27-b9d1-4d9c-a828-748a5c750293)

**Reply by goldn-apk** [reply, marked as answer]
![Screenshot_20240512_105213_Chrome](https://github.com/community/community/assets/169461202/6ec983ea-ce6b-4c02-bb8d-ace28478bbd0)

**Reply by goldn-apk** [reply]
@davevad93 
Thank you

**Reply by davevad93** [reply]
Hey @isbani1 , that means your request has been approved but you still have to wait for your benefits to be available, you will see the following message when you will be able to access them:
 
`Congrats! Your request for @isbani1 was approved and your benefits are now available.`

The waiting time may vary depending on different factors, some people have received their benefits after 2-3 days and others had to wait 9-10 days, so just wait a few more days.

**Reply by davevad93** [reply]
You're welcome @isbani1
